Aptoide/aptoide-client-v8
# Bugfix for navigation crash in screenshots

What does this pull request do?

Fixes a crash in the app view when navigating away from the screen shots view to the app view.


Where should the reviewer start?

- [ ] Go to an app view (like OLX)


How should this be manually tested?

Check if screenshots in the app view carroucel are not stretched.


Tickets related to this pull-request: 

- [ ] https://aptoide.atlassian.net/browse/AN-1823
